redwood-treehouse-host/src/commonMain/kotlin/app/cash/redwood/treehouse/SequentialStateFlow.kt

```diff
@@ -23,15 +23,13 @@ import kotlinx.coroutines.flow.collectLatest
 internal class SequentialStateFlow<T>(
   first: StateFlow<T>,
 ) : StateFlow<T> {
-  public val stateFlowValue: MutableStateFlow<StateFlow<T>> = MutableStateFlow(first)
+  val stateFlowValue: MutableStateFlow<StateFlow<T>> = MutableStateFlow(first)
 
-  override val replayCache: List<T> = listOf()
+  override val replayCache: List<T> = listOf(value)
```

Comment:
Otherwise you always get the initial value
```suggestion
  override val replayCache: List<T> get() = listOf(value)
```